File: Exercicios-Beecrowd/Kotlin/1074.kt
import java.util.*

fun main(args: Array<String>) {
    var contador = readLine().toString().toInt()
    var texto=""
    while (contador!=0){
        texto=""
        contador--
        val numero= readLine().toString().toInt()
        if (numero%2==0){
            texto+="EVEN "

        }
        else{
            texto+="ODD "
        }
        if (numero>0){
            texto+="POSITIVE"
        }
        else{
            texto+="NEGATIVE"
        }
        if (numero==0){
            texto="NULL"
        }

        println(texto)

    }

	
}

File: Exercicios-Beecrowd/Python/1074.py
contador=int(input())
teste=0
while contador != teste:
    x=int(input())
    if x==0:
        print('NULL')
    
    elif x %2==0:
        if x > 0:
            print('EVEN POSITIVE')
        else:
            print('EVEN NEGATIVE')
    else:
        if x>0:
            print('ODD POSITIVE') 
        else:
            print('ODD NEGATIVE')
    teste+=1